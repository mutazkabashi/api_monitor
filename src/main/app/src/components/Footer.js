import React from 'react'
import Card from 'react-bootstrap/Card'; //*Import React-Bootstrap

export default function Footer() {
    const footer = {
        position: 'fixed',
        left: '0',
        bottom: '0',
        width: '100%',
        backgroundColor: '#007bff',
        color: 'white',
        textAlign: 'center'
    }
    return (
        <Card className="text-center" style={footer} >
            <Card.Header>API Monitor &copy; 2021-22</Card.Header>
        </Card>
    )
}
