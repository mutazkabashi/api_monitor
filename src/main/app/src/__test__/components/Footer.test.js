import { render, screen } from '@testing-library/react';
import { shallow } from 'enzyme';

import Footer from '../../components/Footer';

test("renders without crashing", () => {
  const component = shallow(<Footer />);
  // then
  expect(component.getElements()).toMatchSnapshot();
});
