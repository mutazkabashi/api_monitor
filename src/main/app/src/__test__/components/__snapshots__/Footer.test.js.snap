// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <Card
    body={false}
    className="text-center"
    style={
      Object {
        "backgroundColor": "#007bff",
        "bottom": "0",
        "color": "white",
        "left": "0",
        "position": "fixed",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    <CardHeader>
      API Monitor Â© 2021-22
    </CardHeader>
  </Card>,
]
`;
